src/components/SupportPage/index.jsx

```diff
@@ -1,23 +1,68 @@
 import React from 'react';
-import { Helmet } from 'react-helmet';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
 import { MailtoLink } from '@edx/paragon';
 
-import H1 from '../../components/H1';
-
-const SupportPage = () => (
-  <main role="main">
-    <div className="container-fluid mt-3">
-      <Helmet>
-        <title>Support</title>
-      </Helmet>
-      <H1>Support</H1>
-      <p>
-        For assistance, please contact edX Enterprise Support at
-        {' '}
-        <MailtoLink to="customersuccess@edx.org">customersuccess@edx.org</MailtoLink>.
-      </p>
-    </div>
-  </main>
-);
+import SupportForm from './SupportForm';
+import Hero from '../Hero';
+import { features } from '../../config/index';
+
+import './SupportPage.scss';
+
+class SupportPage extends React.Component {
+  hasEmailAndEnterpriseName() {
+    const { emailAddress, enterpriseName } = this.props;
+    return !!(emailAddress && enterpriseName);
+  }
+
+  render() {
+    const {
+      emailAddress,
+      enterpriseName,
+      match,
+    } = this.props;
+
+    return (
+      <React.Fragment>
+        <Helmet>
+          <title>Contact Support</title>
+        </Helmet>
+        <Hero title="Contact Support" />
+        <div className="container-fluid">
+          <div className="row my-3">
+            <div className="col">
+              {features.SUPPORT && this.hasEmailAndEnterpriseName() ? (
+                <SupportForm
+                  initialValues={{
```

Comment:
Looks like you're passing in these `initialValues` (which makes sense) but not using them within `SupportForm`. I think you'll want to use the `initialValues` prop to initialize the form fields with these default values. That way, the `emailAddress` and `enterpriseName` fields will already be filled out.